device/mellanox/x86_64-mlnx_msn2700-r0/pmon_daemon_control.json

```diff
@@ -1,5 +1,6 @@
 {
     "skip_ledd": true,
-    "skip_fancontrol": true
+    "skip_fancontrol": true,
+    "delay_xcvrd": true
```

Comment:
Should this be configurable, or should we simply always delay the xcvrd start?

Reply:
IMO we should simply always delay, like it is done with counters.

Reply:
I heard that there are vendors that depend on the xcvrd to finish their fast-reboot, I am not so sure about that, I was not in that loop before. But if there is no vendor need that, we can simply always delay it. @jleveque any suggestion?

Reply:
I am not familiar with the requirements of fast-reboot on various platforms. @yxieca, @lguohan: are you aware of any platforms which depend on xcvrd to finish fast-reboot?

Reply:
@jleveque As we discussed last time, besides of just monitoring xcvrd dynamically tunes transceivers on some platforms. Thus, if not, on fast-reboot there might be port operationally down longer time.
Guohan can correct me if this is the case.

Reply:
@lguohan could you please share your view on this? Thanks!

Reply:
@jleveque , it is customizable per platform, therefore it is ok. for mostly for the platform i think delay is not used.

Reply:
@lguohan: My question is do we *need* to make this customizable, or should we simply delay it on all platforms? If delaying on all platforms will cause no harm with fast-reboot, then I think it should be delayed on all platforms to make things more universal and also to prevent adding extra complexity.

Reply:
@jleveque In our platforms xcvrd is used to program pre-emphasis and thus a delay would might even lead to ports not coming up. Moreover ins 400G platforms complex initializations need to be done to bring port up and delaying xcvrd there would affect fastboot. Hence we prefer having a platform knob and by default there should be no delay.

Reply:
Thanks, @dgsudharsan. Then we will proceed with this approach, adding a new `delay_xcvrd` parameter.

Reply:
does this means we can sign off it and have it taken to 201911 as well?

Reply:
I have signed off.